tests/snuba/api/endpoints/test_organization_tagkey_values.py

```diff
@@ -555,3 +559,95 @@ def test_invalid_keys(self):
+            mock_replay(
+                seq1_timestamp,
+                self.project.id,
+                replay1_id,
+                urls=[
+                    "http://localhost:3000/",
+                    "http://localhost:3000/login",
+                ],  # duplicate urls are okay,
+                tags={"fruit": "orange"},
+                segment_id=1,
+            ),
+        )
+        self.store_replays(
+            mock_replay(
+                seq1_timestamp,
+                self.project.id,
+                replay2_id,
+                urls=[
+                    "http://localhost:3000/",
+                    "http://localhost:3000/login",
+                ],  # duplicate urls are okay,
+                tags={"fruit": "orange"},
+            )
+        )
+        self.store_replays(
+            mock_replay(
+                seq1_timestamp,
+                self.project.id,
+                replay3_id,
+                urls=[
+                    "http://localhost:3000/",
+                    "http://localhost:3000/login",
+                ],
+                tags={"fruit": "apple", "drink": "water"},
+            )
+        )
+
+    def run_test(self, key, expected, **kwargs):
+        # all tests here require that we search in replays so make that the default here
+        qs_params = kwargs.get("qs_params", {})
+        qs_params["includeReplays"] = "1"
+        kwargs["qs_params"] = qs_params
+        super().run_test(key, expected, **kwargs)
+
+    def test_simple(self):
+        self.run_test("fruit", expected=[("orange", 2), ("apple", 1)])
+        self.run_test("replay_type", expected=[("session", 3)])
+        self.run_test("environment", expected=[("production", 3)])
+        self.run_test("dist", expected=[("abc123", 3)])
+        self.run_test(
+            "url", expected=[("http://localhost:3000/login", 3), ("http://localhost:3000/", 3)]
+        )
+        self.run_test("platform", expected=[("javascript", 3)])
+        self.run_test("release", expected=[("version@1.3", 3)])
+        self.run_test("user.id", expected=[("123", 3)])
+        self.run_test("user.username", expected=[("username", 3)])
+        self.run_test("user.email", expected=[("username@example.com", 3)])
+        self.run_test("user.ip", expected=[("127.0.0.1", 3)])
+        self.run_test("sdk.name", expected=[("sentry.javascript.react", 3)])
+        self.run_test("sdk.version", expected=[("6.18.1", 3)])
+        self.run_test("os.name", expected=[("iOS", 3)])
+        self.run_test("os.version", expected=[("16.2", 3)])
+        self.run_test("browser.name", expected=[("Chrome", 3)])
+        self.run_test("browser.version", expected=[("103.0.38", 3)])
+        self.run_test("device.name", expected=[("iPhone 13 Pro", 3)])
+        self.run_test("device.brand", expected=[("Apple", 3)])
+        self.run_test("device.family", expected=[("iPhone", 3)])
+        self.run_test("device.model_id", expected=[("13 Pro", 3)])
```

Comment:
Can we add mocks and coverage for `first_seen` and `last_seen` values?  Can we also add a generic assert on the schema structure?